Added --enable-metrics support in srt-xtransmit (#39)
mbakholdina/lib-srt-utils: README.md

- Wait for the specified time to have SRT sender and receiver connected and SRT streaming finished,

> - Collect the experiment artifacts (SRT `.csv` statistics, `tshark` `.pcapng` dumps) to the machine where the script is running.

Reviewer: ```suggestion
- Collect the experiment artifacts (SRT `.csv` statistics, `tshark` `.pcapng` dumps) on the machine running the script.
```

> Running the tasks is implemented both locally and remotely so that the following combinations are possible: local-local, local-remote and remote-remote setup. All the implemented configs as well as the detailed documentation can be found in [configs](https://github.com/mbakholdina/lib-srt-utils/tree/master/configs) folder. [Here](https://github.com/mbakholdina/lib-srt-utils/blob/master/configs/rere_xtransmit_live_duration.json) is an example of the remote-remote config where `tshark` and `srt-xtransmit` applications are started consequtively on 2 remote machines (virtual machines in the cloud or regular machines), the SRT transmission is happening and once finished the experiment artifacts are collected to the machine where the script is running.

Reviewer: ```suggestion
Running of the tasks is implemented both locally and remotely so that the following combinations are possible: local-local, local-remote, and remote-remote setup. All the implemented configs as well as the detailed documentation can be found in the [configs](https://github.com/mbakholdina/lib-srt-utils/tree/master/configs) folder. [Here](https://github.com/mbakholdina/lib-srt-utils/blob/master/configs/rere_xtransmit_live_duration.json) is an example of SRT transmission in a remote-remote config where `tshark` and `srt-xtransmit` applications are started consecutively on 2 remote machines (virtual machines in the cloud or regular machines). Once the run is finished the experiment artifacts are collected on the machine running the script.
```
